scripts/lstchain_data_create_pedestal_file.py

```diff
@@ -15,24 +15,35 @@
 To run script in console:
 python lstchain_data_create_pedestal_file.py --input_file LST-1.1.Run00097.0000.fits.fz --output_file pedestal.fits 
 --max_events 9000
+To set start sample in waveform --start_r0_waveform i (default i = 11)
 not to use deltaT correction add --deltaT False
 '''
 
 parser = argparse.ArgumentParser()
 
 # Required arguments
-parser.add_argument("--input_file", help="Path to fitz.fz file to create pedestal file.",
-                    type=str, default="")
+parser.add_argument("--input_file",
+                    help="Path to fitz.fz file to create pedestal file.",
+                    type=str,
+                    default="")
 
-parser.add_argument("--output_file", help="Path where script create pedestal file",
+parser.add_argument("--output_file",
+                    help="Path where script create pedestal file",
                     type=str)
 
 # Optional argument
-parser.add_argument("--max_events", help="Maximum numbers of events to read."
-                                         "Default = 5000",
-                    type=int, default=5000)
-
-parser.add_argument('--deltaT', '-s', type=lambda x: bool(strtobool(x)),
+parser.add_argument("--max_events",
+                    help="Maximum numbers of events to read. Default = 5000",
+                    type=int,
+                    default=5000)
+
+parser.add_argument("--start_r0_waveform",
+                    help="Start sample for waveform. Default = 10",
```

Comment:
default in the variable definition and the description are not coincident

Reply:
Yes, I have fixed